EventFilter/DTRawToDigi/python/dtunpackerDDUGlobal_cfi.py

```diff
@@ -22,3 +22,9 @@
     localDAQ = cms.untracked.bool(False),
     performDataIntegrityMonitor = cms.untracked.bool(True)
 )
+
+import EventFilter.DTRawToDigi.dturosunpacker_cfi
+_dturosunpacker = EventFilter.DTRawToDigi.dturosunpacker_cfi.dturosunpacker.clone()
+from Configuration.Eras.Modifier_run2_DT_2018_cff import run2_DT_2018
+run2_DT_2018.toReplaceWith(dtunpacker, _dturosunpacker)
```

Comment:
```dtunpacker``` is already era-customized in its own configuration:
EventFilter/DTRawToDigi/python/dtunpacker_cfi.py
No need to repeat it in every config in which it is imported